server/src/main/resources/reference.conf

```
@@ -10,6 +10,10 @@
     # maximum number of connections that HikariCP will keep in the pool, including both idle and in-use connections
     maxPoolSize=10
   }
+  aws {
+    region = "af-south-1"
+    serviceUserSecretKey = "serviceUserSecretKey"
```

I would definitely mention DB here. This is for secret for accessing database

```suggestion
    awsSecretNameForDBServiceUserPwd = "changeme"
```

or something like that

---

I was actually not entirely sure I like this to be here, as I think that it's closer to the DB. But I understand that we might have more AWS stuff and it can belong close to each other for having better visibility of what's used from AWS...but hmm on the other hand, if working with the DB, such config-using code would be using other unrelated things, such as config for SSM or so...

I guess I'm fine with this currently

---

You are right, it wasn't obvious it's related to a database. And as you mentioned, it's difficult to foresee how things will evolve, what sort of AWS services we will be using. And even then we will face the dilemma of whether to group it by service or by technology. Anyway I have change the parameter to **dbPasswordSecretName** as I think it makes it clear and still isn't crazy long. Hope that's acceptable.

---

it is, thanks it looks  better :)